rpcs3/Emu/Cell/Modules/cellSysutil.cpp

```diff
@@ -82,19 +110,22 @@ extern s32 sysutil_send_system_cmd(u64 status, u64 param)
 			}
 		}
 
-		for (sysutil_cb_manager::registered_cb cb : cbm->callbacks)
```

Comment:
Not a bug, cb is not a reference it cannot become nullptr afterwards.

Reply:
Hmm. The motivation was to fix a logic race where the callback may still be executed after UnregisterCallback.
But that would have to happen in CheckCallback as well.

Reply:
I'm currently afk, but you need to check how it works in disassembly. It can work the way you said, it can also work by trying to register 4 distinct callbacks each registeration. In the latter case the current master is right. All callbacks are registered through cellSysutilRegisterCallbackDispatcher as far as I know so I think it is the latter case but I cannot be certian.

Reply:
removed the commit